### PR TITLE
Bump journald-logger, journad-native and logging-journald

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly

---

### CODE REVIEW

**Comment by lzap** [comment]
Looks like @ekohl realized I forgot to change `rel-eng/tito.props` in my previous PR. This sorts this out, also it adds `rubygem-logging-journald` into non-SCL group as well, which I forgot to add. That is the gem which uses the "journald-logging" and "journald-native" we need to have.

**Comment by lzap** [comment]
Oh one more:

```
DEBUG util.py:491:  Getting requirements for rubygem-journald-logger-2.0.4-1.el7.src
DEBUG util.py:491:   --> ruby-libs-2.0.0.648-33.el7_4.x86_64
DEBUG util.py:491:   --> rubygems-devel-2.0.14.1-33.el7_4.noarch
DEBUG util.py:491:  Error: No Package found for ruby >= 2.1.0
```

**Comment by lzap** [comment]
@ekohl packages in koji are passing, but the "rpm" task has no results. Not sure what this means tho.

**Comment by lzap** [comment]
Ok I have realized that I actually need to bump release or version of all the three RPMs to get them rebuilt. Therefore in this PR I am going to:

* make sure that journald-logger, journad-native and logging-journald are all in non-SCL group
* bump version of journald-logger to Ruby 2.0 compatible version
* bump version of journald-native to latest version
* bump version of logging-journald to latest version

This is now WIP

**Comment by lzap** [comment]
This is now ready! I tested this on nightly instance downloading the scratchbuilds from the last run:

```
[root@nightly ~]# yum install *rpm
Loaded plugins: fastestmirror
Examining rubygem-journald-logger-2.0.4-1.el7.noarch.rpm: rubygem-journald-logger-2.0.4-1.el7.noarch
Marking rubygem-journald-logger-2.0.4-1.el7.noarch.rpm to be installed
Examining rubygem-journald-logger-doc-2.0.4-1.el7.noarch.rpm: rubygem-journald-logger-doc-2.0.4-1.el7.noarch
Marking rubygem-journald-logger-doc-2.0.4-1.el7.noarch.rpm to be installed
Examining rubygem-journald-native-1.0.11-1.el7.x86_64.rpm: rubygem-journald-native-1.0.11-1.el7.x86_64
Marking rubygem-journald-native-1.0.11-1.el7.x86_64.rpm to be installed
Examining rubygem-journald-native-doc-1.0.11-1.el7.noarch.rpm: rubygem-journald-native-doc-1.0.11-1.el7.noarch
Marking rubygem-journald-native-doc-1.0.11-1.el7.noarch.rpm to be installed
Examining rubygem-logging-journald-2.0.0-1.el7.noarch.rpm: rubygem-logging-journald-2.0.0-1.el7.noarch
Marking rubygem-logging-journald-2.0.0-1.el7.noarch.rpm to be installed
Examining rubygem-logging-journald-doc-2.0.0-1.el7.noarch.rpm: rubygem-logging-journald-doc-2.0.0-1.el7.noarch
Marking rubygem-logging-journald-doc-2.0.0-1.el7.noarch.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package rubygem-journald-logger.noarch 0:2.0.4-1.el7 will be installed
---> Package rubygem-journald-logger-doc.noarch 0:2.0.4-1.el7 will be installed
---> Package rubygem-journald-native.x86_64 0:1.0.11-1.el7 will be installed
---> Package rubygem-journald-native-doc.noarch 0:1.0.11-1.el7 will be installed
---> Package rubygem-logging-journald.noarch 0:2.0.0-1.el7 will be installed
---> Package rubygem-logging-journald-doc.noarch 0:2.0.0-1.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

===============================================================================================================================
 Package                           Arch        Version             Repository                                             Size
===============================================================================================================================
Installing:
 rubygem-journald-logger           noarch      2.0.4-1.el7         /rubygem-journald-logger-2.0.4-1.el7.noarch            11 k
 rubygem-journald-logger-doc       noarch      2.0.4-1.el7         /rubygem-journald-logger-doc-2.0.4-1.el7.noarch       243 k
 rubygem-journald-native           x86_64      1.0.11-1.el7        /rubygem-journald-native-1.0.11-1.el7.x86_64           25 k
 rubygem-journald-native-doc       noarch      1.0.11-1.el7        /rubygem-journald-native-doc-1.0.11-1.el7.noarch      150 k
 rubygem-logging-journald          noarch      2.0.0-1.el7         /rubygem-logging-journald-2.0.0-1.el7.noarch          5.9 k
 rubygem-logging-journald-doc      noarch      2.0.0-1.el7         /rubygem-logging-journald-doc-2.0.0-1.el7.noarch      208 k

Transaction Summary
===============================================================================================================================
Install  6 Packages

Total size: 641 k
Installed size: 641 k
Is this ok [y/d/N]: y
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : rubygem-journald-native-1.0.11-1.el7.x86_64                                                                 1/6 
  Installing : rubygem-journald-logger-2.0.4-1.el7.noarch                                                                  2/6 
  Installing : rubygem-logging-journald-2.0.0-1.el7.noarch                                                                 3/6 
  Installing : rubygem-logging-journald-doc-2.0.0-1.el7.noarch                                                             4/6 
  Installing : rubygem-journald-logger-doc-2.0.4-1.el7.noarch                                                              5/6 
  Installing : rubygem-journald-native-doc-1.0.11-1.el7.noarch                                                             6/6 
  Verifying  : rubygem-logging-journald-2.0.0-1.el7.noarch                                                                 1/6 
  Verifying  : rubygem-logging-journald-doc-2.0.0-1.el7.noarch                                                             2/6 
  Verifying  : rubygem-journald-logger-doc-2.0.4-1.el7.noarch                                                              3/6 
  Verifying  : rubygem-journald-native-doc-1.0.11-1.el7.noarch                                                             4/6 
  Verifying  : rubygem-journald-native-1.0.11-1.el7.x86_64                                                                 5/6 
  Verifying  : rubygem-journald-logger-2.0.4-1.el7.noarch                                                                  6/6 

Installed:
  rubygem-journald-logger.noarch 0:2.0.4-1.el7                 rubygem-journald-logger-doc.noarch 0:2.0.4-1.el7                
  rubygem-journald-native.x86_64 0:1.0.11-1.el7                rubygem-journald-native-doc.noarch 0:1.0.11-1.el7               
  rubygem-logging-journald.noarch 0:2.0.0-1.el7                rubygem-logging-journald-doc.noarch 0:2.0.0-1.el7               

Complete!

[root@nightly ~]# irb
irb(main):001:0> require 'logging'
=> true
irb(main):002:0> 
irb(main):003:0* log = Logging.logger['app']
=> #<Logging::Logger:0x000000025f5170
... cut ...
irb(main):007:0* Logging.mdc['signed_user'] = 'Ondra'
=> "Ondra"
irb(main):009:0> log.debug "blah blah"
=> true
irb(main):010:0> 

[root@nightly ~]# journalctl -e | tail -n4
Oct 12 12:52:29 nightly.nat.lan yum[7290]: Installed: rubygem-logging-journald-doc-2.0.0-1.el7.noarch
Oct 12 12:52:30 nightly.nat.lan yum[7290]: Installed: rubygem-journald-logger-doc-2.0.4-1.el7.noarch
Oct 12 12:52:31 nightly.nat.lan yum[7290]: Installed: rubygem-journald-native-doc-1.0.11-1.el7.noarch
Oct 12 12:53:46 nightly.nat.lan myservice[7314]: blah blah
```

File: packages/foreman/rubygem-journald-native/rubygem-journald-native.spec [hunk 1]
```diff
@@ -62,32 +62,34 @@ cp -pa .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 mkdir -p %{buildroot}%{gem_extdir_mri}
-cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+%if 0%{?scl:1}
+  cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+%else
+  cp -a ./usr/share/gems/gems/%{gem_name}-%{version}/ext/journald_native/journald_native.so %{buildroot}%{gem_extdir_mri}/
```
Comment (ekohl) [inline packages/foreman/rubygem-journald-native/rubygem-journald-native.spec]: Using `%{gem_instdir}` should be equal to `/usr/share/gems/gems/%{gem_name}-%{version}`

File: packages/foreman/rubygem-journald-logger/rubygem-journald-logger.spec [hunk 2]
```diff
@@ -76,10 +76,14 @@ cp -pa .%{gem_dir}/* \
...
 %doc %{gem_instdir}/README.md
+%doc %{gem_instdir}/CHANGELOG.md
 %{gem_instdir}/Rakefile
 
 %changelog
-* Wed Sep 05 2018 Eric D. Helms <ericdhelms@gmail.com> - 2.0.3-2
+* Thu Oct 11 2018 Lukas Zapletal <lzap+rpm@redhat.com> 2.0.4-1
+- Bumped version for Ruby 2.0 support
+
+* Wed Sep 05 2018 Eric D. Helms <ericdhelms@gmail.com> 2.0.3-2
```
Comment (ekohl) [inline packages/foreman/rubygem-journald-logger/rubygem-journald-logger.spec]: Nit: I'd appreciate keeping the `- ` in Erics changelog to have a minimal change.

File: packages/foreman/rubygem-journald-native/rubygem-journald-native.spec (hunk 1 above)
Comment (lzap) [reply packages/foreman/rubygem-journald-native/rubygem-journald-native.spec]: Sure.

**Comment by lzap** [comment]
Rebased, all green.

**Comment by lzap** [comment]
@ekohl or @ehelms I believe this is 99% correct :-)

**Pull request merged**